### PR TITLE
Add content language switcher

### PR DESCRIPTION
**Pull request by timoludwig**
### Short description

This adds a better content language switcher:

Collapsed             |  Expanded
:-------------------------:|:-------------------------:
Screenshot_2021-05-09 Integreat Redaktionssystem |  Screenshot_2021-05-09 Integreat Redaktionssystem(1)

### Proposed changes

- Add flackpack for svg flags
- Add constants for countries
- Add two possible countries per language
- Add language switcher

### Resolved issues

Fixes: #630

### CODE REVIEW

**Comment by svenseeberg** [comment]
I noticed one GUI glitch with the drop down:
myimage

**Comment by svenseeberg** [comment]
It would be easiest to merge #818 into this branch before merging into the develop branch.

**Comment by ulliholtgrave** [comment]
I have the same issue, but I think it's not that annoying. If we really have to, we might add a extra div around the chevron that keeps the size.

File: src/cms/constants/countries.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    ("vn", _("Viet Nam")),
+    ("xk", _("Kosovo")),
+    ("ye", _("Yemen")),
+    ("za", _("South Africa")),
+    ("zm", _("Zambia")),
+    ("zw", _("Zimbabwe")),
+)
+
+#: Choices which are currently inactive and cannot be used to represent languages
+INCATIVE_CHOICES = (
```
Comment (svenseeberg) [inline src/cms/constants/countries.py:185]: `INACTIVE_CHOICES` ?

**Review by svenseeberg — CHANGES_REQUESTED** [review]
I have not found any major issues. It works with a minor GUI glitch in my Firefox. I think fixing the GUI glitch could be part of a bug fix later on. There is one typo in the inactive constants.

**Review by svenseeberg — APPROVED** [review]
I wrote my last comment before I read that you already fixed it. So yes, good to merge!

**Pull request merged**